src/main/java/com/strangeone101/bendinggui/menus/MenuEditElements.java

```diff
@@ -113,22 +113,28 @@ public void onClick(Player playerwhoclicked)
 				{
 					if (playerwhoclicked.hasPermission("bending.admin.add"))
 					{
-						if (player instanceof Player)
-							((Player)player).sendMessage(ChatColor.YELLOW + "You are now " + (element == Element.AIR || element == Element.EARTH ? "an " : "a ") + c + element.getName().toLowerCase() + ChatColor.YELLOW + " " + element.getType().getBender() + "!");
+						if (player.isOnline())
```

Comment:
Why did you change this? This actually makes the cast bellow technically unsafe.